doc/DOWNLOADS.md

```diff
@@ -60,33 +62,33 @@ pkg install py-nicotine-plus
 
 ### pip
 
-If Nicotine+ is not available for your system, it can be installed using [pip](https://pip.pypa.io/en/stable/). Ensure the [runtime dependencies](DEPENDENCIES.md) are installed, and run the following:
+If Nicotine+ is not packaged for your system, the current stable version can be installed using [pip](https://pip.pypa.io/). Ensure the [runtime dependencies](DEPENDENCIES.md) are installed, and run the following:
 
 ```sh
 pip3 install nicotine-plus
 ```
 
-Keep in mind that Nicotine+ will not auto-update. When a new release is available, run the following:
+Keep in mind that Nicotine+ will not update automatically. When a new release is available, run the following:
 
 ```sh
 pip3 install --upgrade nicotine-plus
 ```
 
 ## Windows
 
-### Official Builds
+### Official Release
 
 Stable Windows installers for Nicotine+ are available for download. Installing Nicotine+ requires administrator privileges.
 
-NOTE: The installer format has changed in Nicotine+ 3.2.0. If you are upgrading from Nicotine+ 3.1.1 or earlier, please uninstall Nicotine+ first (this will not remove your existing settings).
+*NOTE: The installer format has changed since Nicotine+ 3.2.0. If you are upgrading from Nicotine+ 3.1.1 or earlier, please uninstall Nicotine+ first (this will not remove your existing settings).*
 
-- [64-bit Installer](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-installer.zip.sha256)]
-- [32-bit Installer](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-i686-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-i686-installer.zip.sha256)]
+- [Download Nicotine+ 64-bit Windows Installer *(current stable version)*](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-installer.zip.sha256)]
```

Comment:
Do we need this verbosity? These links are rather awkward to look at and read at this length:
![Screenshot from 2022-02-02 21-23-44](https://user-images.githubusercontent.com/8754153/152222473-009c539f-9426-4970-a8ec-3482fc839dc4.png)

I think including "Download" in the original text would be fine, e.g. "Download 64-bit Installer".

Reply:
But then there would nothing to index with the Windows keyword on the SERP's.

I suggest we don't need _(current stable version)_ that is what causes the awkward appearance.